docs/support_metrics.md

```diff
@@ -0,0 +1,30 @@
+# Support Metrics
+
+Here defines all the support metrics info of Harbor.
+
+## Replication Adapter
+
+|     | Registries       | Pull Mode | Push Mode |
+|-----|------------------|-----------|-----------|
+| Harbor|  ![Harbor](img/replication_adapters/harbor_logo.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)|
+| distribution | ![distribution](img/replication_adapters/distribution.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)|
+| docker hub | ![docker hub](img/replication_adapters/docker_hub.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)|
+| GCR | ![GCR](img/replication_adapters/gcr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)|
+| ECR | ![ECR](img/replication_adapters/ecr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)|
+| ACR | ![ACR](img/replication_adapters/acr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)|
+| AliCR | ![AliCR](img/replication_adapters/ali-cr.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)|
+| Helm Hub | ![Helm Hub](img/replication_adapters/helm-hub.png)|![Y](img/replication_adapters/right.png)| N/A |
+| Huawei SWR | ![Huawei SWR](img/replication_adapters/hw.png)|![Y](img/replication_adapters/right.png)|![Y](img/replication_adapters/right.png)|
+
+**Notes**: Pull mode means pull content from the specified source registries into harbor. Push mode means push the content of Harbor to the specified target registries.
```

Comment:
minor adjustments -> 'Pull mode is replicating artifacts from specified source registries into Harbor.  Push mode is replicating artifacts from Harbor to specified target registries'